argoverse/__init__.py

```diff
@@ -0,0 +1 @@
+VERSION = "1.1.0"
```

Comment:
Used in `setup.cfg`.

Reply:
is the version required to be placed here? This probably wouldnt be a place I would look for the version number, so do we need to write it in 2 places now?

Reply:
This is the first method listed here: https://packaging.python.org/guides/single-sourcing-package-version/#single-sourcing-the-package-version. Although, they use `__version__` instead of `VERSION`.

Reply:
Updated to use `__version__`.